CMakeLists.txt

```diff
@@ -157,7 +157,7 @@ endif()
 no_warning(enum-constexpr-conversion) # breaks Protobuf in clang-16
 
 option(ENABLE_TESTS "Provide unit_test_dbms target with Google.Test unit tests" ON)
-option(ENABLE_EXAMPLES "Build all example programs in 'examples' subdirectories" OFF)
+option(ENABLE_EXAMPLES "Build all example programs in 'examples' subdirectories" ON)
```

Comment:
Let's disable compilation of tests by default.

Reply:
Yes, I was leaving it on purpose to make sure that the GCP KMS example builds and links on the CI. Once I check that, I'll revert the commit that adds the source code and builds examples by default